src/assets/styles/colors.scss

```diff
@@ -16,4 +16,7 @@ $success-background: rgb(226, 254, 229);
 $success-text: rgb(29, 63, 37);
 
 $warning-background: rgb(135, 155, 85);
-$warning-text: rgb(255, 255, 101);
\ No newline at end of file
+$warning-text: rgb(255, 255, 101);
+
+$amount-expence: red;
```

Comment:
It is `expense` not `expence`, also lets go with HEX or RGB values. I would maybe go with RGB/RGBA for all color values in this file.